Library/Formula/lua.rb

```diff
@@ -114,7 +117,7 @@ def install
   def pc_file; <<-EOS.undent
     V= 5.2
     R= 5.2.4
-    prefix=#{HOMEBREW_PREFIX}
+    prefix=#{prefix}
```

Comment:
Yeah. I don't really have a problem with `prefix` but I am wary we're one of the few package managers off the top of my head to set `pkg-config` paths pointing at something that isn't the top-level directory, thanks to our unusual symlink from the Cellar structure.

I think there's probably a discussion to ponder a bit more there, but I don't mind `prefix` staying as `prefix` until we make a wider decision.